.gitlab/tests.yml

```diff
@@ -40,7 +41,6 @@ include:
   variables:
     CMAKE_BUILD_PARALLEL_LEVEL = "12"
     CARGO_BUILD_JOBS = "12"
-    DD_FAST_BUILD = "1"
```

Comment:
did you want to add a GitLab cache for the pip cache?

Reply:
Would it be enough to set `PIP_CACHE_DIR` to `'${CI_PROJECT_DIR}/.cache/pip' in `hatch.toml::[envs.appsec_iast_packages.env-vars]`? (I will try)

Reply:
that config setup in `gen_gitlab_config.py` is only for `build_base_venvs` it isn't shared across all jobs.

you need to update the job you care about to also use a cache like that.

also, that is `${CI_PROJECT_DIR}/.cache`, you cannot save/restore anything outside of `${CI_PROJECT_DIR}` (citation needed)